x/tokenfactory/keeper/genesis_test.go

```diff
@@ -32,6 +32,9 @@ func (suite *KeeperTestSuite) TestGenesis() {
 		},
 	}
 	app := suite.App
+	// remove module account to ensure initGenesis initializes it on its own
+	tokenfactoryModuleAccount := app.AccountKeeper.GetAccount(suite.Ctx, app.AccountKeeper.GetModuleAddress(types.ModuleName))
+	app.AccountKeeper.RemoveAccount(suite.Ctx, tokenfactoryModuleAccount)
```

Comment:
If we don't remove the module account here, we are never actually testing if initGenesis creates the module account properly

Reply:
We should make a new suite.SetupGenesisTest() constructor in the future, to not have this be a concern in the future

Reply:
Issue created! https://github.com/osmosis-labs/osmosis/issues/2502